### DISCUSSION TITLE
Background image opacity

### DISCUSSION THREAD

**Discussion by TaylorJonesTRT** [wezterm/wezterm#5876] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Is there a way to change the opacity of ONLY the background image? 
I have the below in my config but the opacity flag/option is changing the opacity of the entire window and not just the image:
```config.background = {
	{
		source = {
			File = "/Users/taylorjones/powder.jpg",
		},
		opacity = 0.9,
	},
}
```

**Comment by nekowinston** [comment]
If what you're looking for is simply making the background less bright, you can use the `hsb` option in the layer definition, e.g.:
```lua
config.background = {
	{
		source = {
			File = wezterm.home_dir .. "/Pictures/Wallpapers/mountainlion.jpg",
		},
		hsb = { brightness = 0.05 },
	},
}
```

Which will reduce the brightness of the picture by 95%.

**Reply by TaylorJonesTRT** [reply]
> If what you're looking for is simply making the background less bright, you can use the `hsb` option in the layer definition, e.g.:
> 
> ```lua
> config.background = {
> 	{
> 		source = {
> 			File = wezterm.home_dir .. "/Pictures/Wallpapers/mountainlion.jpg",
> 		},
> 		hsb = { brightness = 0.05 },
> 	},
> }
> ```
> 
> Which will reduce the brightness of the picture by 95%.

This sort of works but I'm more looking for a way to blend the image into the current color schemes background color.

**Reply by nekowinston** [reply, marked as answer]
Maybe
```lua
c.background = {
	{
		source = {
			File = wezterm.home_dir .. "/Pictures/Wallpapers/mountainlion.jpg",
		},
	},
	{
		source = {
			Color = "rgba(28, 33, 39, 0.95)",
		},
		height = "100%",
		width = "100%",
	},
}
```

**Reply by TaylorJonesTRT** [reply]
> Maybe
> 
> ```lua
> c.background = {
> 	{
> 		source = {
> 			File = wezterm.home_dir .. "/Pictures/Wallpapers/mountainlion.jpg",
> 		},
> 	},
> 	{
> 		source = {
> 			Color = "rgba(28, 33, 39, 0.95)",
> 		},
> 		height = "100%",
> 		width = "100%",
> 	},
> }
> ```

This worked perfectly!!

**Comment by bro69noscope** [comment]
`Color = "rgba(28, 33, 39, 0.95)",`

Oh ! Wow, took me an hour to get here, how did you guys figure this out I cannot find this anywhere in the docs, even now I'm looking I'm still not finding it...

**Comment by JustBarnt** [comment]
I am also curious did I misread something on why I would need to set a `width` and `height` on my color in order for it to take effect?
